spec/lib/importers/view_importer_spec.rb

```diff
@@ -28,34 +28,39 @@
   end
 
   describe '.update_all_views' do
+    let!(:course) do
+      create(:course, id: 10001,
```

Comment:
Similar code found in view_importer_spec.rb:15, view_importer_spec.rb:32, view_importer_spec.rb:79